libethereum/State.cpp

```diff
@@ -367,17 +376,23 @@ void State::subBalance(Address const& _id, bigint const& _amount)
 
 void State::createContract(Address const& _address, bool _incrementNonce)
 {
-	m_cache[_address] = Account(
+	createInCache(_address, Account(
 		requireAccountStartNonce() + (_incrementNonce ? 1 : 0),
 		balance(_address),
-		Account::ContractConception
+		Account::ContractConception)
 	);
 }
 
 void State::ensureAccountExists(const Address& _address)
 {
 	if (!addressInUse(_address))
-		m_cache[_address] = Account(requireAccountStartNonce(), 0, Account::NormalCreation);
+		createInCache(_address, Account(requireAccountStartNonce(), 0, Account::NormalCreation));
+}
+
+void State::createInCache(Address const& _address, Account const&& _account)
```

Comment:
It will be created on disk in the end, so perhaps just call that `createAccount`?